web/themes/custom/recipe_theme/templates/field/field--user--name.html.twig

```diff
@@ -0,0 +1,52 @@
+{#
```

Comment:
This template is for the submitter information?

It's not a field, it probably needs to handled in `node.html.twig`.

https://git.drupalcode.org/project/radix/-/blob/8.x-4.x/templates/content/node.html.twig

Line 28-36